Library/Formula/gitlab-ci-multi-runner.rb

```diff
@@ -16,21 +18,44 @@ class GitlabCiMultiRunner < Formula
   end
 
   depends_on "go" => :build
+  depends_on "godep" => :build
 
-  def install
-    ENV["GOPATH"] = buildpath
+  go_resource "github.com/jteeuwen/go-bindata" do
+    url "https://github.com/jteeuwen/go-bindata.git",
+      :revision => "a0ff2567cfb70903282db057e799fd826784d41d"
+  end
 
+  resource "prebuilt.tar.gz" do
+    url "https://gitlab-ci-multi-runner-downloads.s3.amazonaws.com/v1.0.4/docker/prebuilt.tar.gz",
+      :using => :nounzip
+    sha256 "43dedd023672990e27289e97bf74f493576956418148f6227917b8511e8aadfd"
+  end
```

Comment:
@ReDetection You need prebuilt images to build gitlab-ci-multi-runner, no matter how you use, as the docker feature is not optional for gitlab-ci-multi-runner binary. This is about how you build gitlab-ci-multi-runner

What you can decide is whether using docker executor, to use docker executor you need docker installed. And this is about how you use gitlab-ci-multi-runner